calculator.rb

```diff
@@ -0,0 +1,103 @@
+#simple calculator
+
+#method to list the operators we can use
+def list_options
+  puts "\nThe operator options we have are:"
+  puts "\n1. add(+)\n2. subtract(-)\n3. multiply(*)\n4. divide(/)"
+end
+
+#Method to prompt the user for which operator they would like to use.
+#If the answer the user gives is not a valid operator,
+#prompt again until it is. Offer the option to revisit the list of valid operators.
+def grab_operator
+  print "\nChoose the type of operator you'd like to use(name): "
+  operator = gets.chomp
+  operators = ["add", "+", "subtract", "-", "multiply", "*", "divide", "/"]
+  until operators.include?(operator)
+    puts "\nHmm! It looks like we don't use that type of operator here."
+    puts "To see the types of operators you can use, type \"options\"."
+    print "Otherwise, enter the operator you would like to use: "
+    operator = gets.chomp
+    if operator == "options"
+      list_options()
+    end
+  end
+  return operator
+end
+
+#Method to check whether or a not a value is a number.
+def is_number? (num)
+  if num == num.to_i.to_s
+    return true
+  elsif num[0..2] == num[0..2].to_f.to_s
+    return true
+  else
+    return false
+  end
+end
+
+#Method to prompt the user for two operands.
+def grab_operands
+  operands = []
+  2.times do |i|
+    side = ""
+    if i == 0
+      side = "left"
+    else
+      side = "right"
+    end
```

Comment:
Did you know you could have an array `side = ["left", "right"]` and then use `i` for an index?  That could help compact things.